src/Tgstation.Server.Host/Tgstation.Server.Host.csproj

```diff
@@ -23,19 +23,19 @@
     <DefineConstants>$(DefineConstants);NO_WEBPANEL</DefineConstants>
   </PropertyGroup>
 
-  <Target Condition="'$(TGS_HOST_NO_WEBPANEL)' != 'true'" Name="ClientInstall" BeforeTargets="ResolveAssemblyReferences" Inputs="../../build/WebpanelVersion.props" Outputs="$(NpmInstallStampFile)">
+  <Target Condition="'$(TGS_HOST_NO_WEBPANEL)' != 'true'" Name="ClientInstall" BeforeTargets="ResolveAssemblyReferences" Inputs="../../build/WebpanelVersion.props" Outputs="$(BunInstallStampFile)">
     <Message Text="Pulling web control panel..." Importance="high" />
     <RemoveDir Directories="ClientApp" />
     <Exec Command="git clone https://github.com/tgstation/tgstation-server-webpanel --branch v$(TgsWebpanelVersion) --depth 1 ClientApp" />
     <RemoveDir Directories="ClientApp/.git" /> <!-- Allows git clean to clean it out -->
-    <Message Text="Restoring yarn packages..." Importance="high" />
-    <Exec WorkingDirectory="ClientApp" Command="yarn install --immutable" />
-    <Touch Files="$(NpmInstallStampFile)" AlwaysCreate="true" />
+    <Message Text="Restoring bun packages..." Importance="high" />
+    <Exec WorkingDirectory="ClientApp" Command="bun install --frozen-lockfile" />
```

Comment:
Will this work with a yarn.lock? Which is what https://github.com/tgstation/tgstation-server-webpanel has

Reply:
Looks like I'll need a pr there!